orion/pipelines/verified/tadgan/tadgan_without_dropout_gpu.json

```diff
@@ -0,0 +1,173 @@
+{
+    "primitives": [
+        "mlprimitives.custom.timeseries_preprocessing.time_segments_aggregate",
+        "sklearn.impute.SimpleImputer",
+        "sklearn.preprocessing.MinMaxScaler",
+        "mlprimitives.custom.timeseries_preprocessing.rolling_window_sequences",
+        "orion.primitives.timeseries_preprocessing.slice_array_by_dims",
+        "orion.primitives.tadgan.TadGAN",
+        "orion.primitives.tadgan.score_anomalies",
+        "orion.primitives.timeseries_anomalies.find_anomalies"
+    ],
+    "init_params": {
+        "mlprimitives.custom.timeseries_preprocessing.time_segments_aggregate#1": {
+            "time_column": "timestamp",
+            "interval": 21600,
+            "method": "mean"
+        },
+        "sklearn.preprocessing.MinMaxScaler#1": {
+            "feature_range": [
+                -1,
+                1
+            ]
+        },
+        "mlprimitives.custom.timeseries_preprocessing.rolling_window_sequences#1": {
+            "target_column": 0,
+            "window_size": 100,
```

Comment:
reading this I just realized there is no easy guarantee some hyperparameters across different primitives share the same values. 

I am wondering whether mlblock supports the following feature or not?
```json
{
"mlprimitives.custom.timeseries_preprocessing.rolling_window_sequences#1": {
            "target_column": 0,
            "window_size": "orion.primitives.tadgan.TadGAN#1.input_shape",
            "target_size": 1
},
"orion.primitives.tadgan.TadGAN#1": {
            "epochs": 35,
            "input_shape": [100, 1],
            "...": "..."
}

```

Reply:
This feature is not currently supported as far as I know.

Reply:
Documented issue in sintel-dev channel.